client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -269,14 +269,12 @@
                     <div class="oui-tile__item"
                          data-ng-if="worldPart !== 'US'">
                         <div class="oui-tile__definition">
-                            <div data-working-status
-                                 data-modal-id="'worksDedicatedServers'"
-                                 data-product-type="'dedicated_server'"
-                                 data-works-details="works"
-                                 data-show-affiliated="true"
-                                 data-show-only-active="true"
-                                 data-tr="tr">
-                            </div>
+                            <p data-translate="core_works_description"></p>
+                                <button type="button"
```

Comment:
Why not use `oui-button` ?

Reply:
Also, indentation

Reply:
Well I tried ussing oui-button but it currently lacks a way to go full-width.
We'll add the ability to do that in oui, but in the mean time, it's a normal button.